### ISSUE TITLE
After updating the app can no longer find the Nextcloud files app

### ISSUE THREAD

**Issue by tux93** [UP-NextPush/android#55]
After updating NextPush via F-Droid it reports the Nextcloud Files app is not installed, despite having previously been successfully connected.

The only workaround I have found so far is to clear data of the NextPush app and login again, but this is not desirable as it means any apps that have been using NextPush get unregistered.

NextPush version 1.4.2
Nextcloud version 3.22.1
Both installed via F-Droid

**Comment by p1gp1g** [comment]
What was the previous version of NextPush on android ? 1.4.0 or 1.4.1 ?

**Comment by p1gp1g** [comment]
It has been available for 1 or 2 days but I've asked to disable it since it was buggy

**Comment by tux93** [comment]
In that case I'm fairly certain it was 1.4.0 as I did not update immediately.

On a side note: the same happened on a earlier update from 1.3.3 to 1.4.0 but at the time I put it off as a one time glitch

**Comment by tux93** [comment]
Hmm, out o curiosity I did the following to see if I could reproduce it:

1. Uninstall the current Nextpush
2. Install Nextpush 1.3.3 and log in
3. Update to 1.4.0 -> Error did not appear
4. Update to 1.4.2 -> Error did not appear

So either I really did get unlucky and catch the buggy 1.4.1 initially and that was the whole reason or there is another factor like that when I ran into the problem I updated both Nextpush and the Files app shortly after another.

I'd like to keep the issue open until the next update until a new update for the Files app arrives, if the error does not re-appear then either I'll close it.
